components/payments/internal/connectors/plugins/public/stripe/accounts.go

```diff
@@ -0,0 +1,91 @@
+package stripe
+
+import (
+	"context"
+	"encoding/json"
+	"time"
+
+	"github.com/formancehq/payments/internal/connectors/plugins/currency"
+	"github.com/formancehq/payments/internal/models"
+)
+
+var (
+	rootAccountReference = "root"
+)
+
+// root account reference is internal so we don't pass it to Stripe API clients
+func resolveAccount(ref *string) *string {
```

Comment:
Do you need to pass a pointer here for the reference ?

Reply:
I don't need to. We can change it to a simple string if you prefer.

More just stylistic because the v2 implementation was passing around a potentially nillable account to methods like Translate that behave slightly different depending on whether an account was specified or not. So I am treating `from.Reference` from the cache as something that we expect might be nil across the board (as opposed to having empty string checks).